modules/SecurityGroups/MassAssign.php

```diff
@@ -47,6 +47,13 @@
     } else {
         $where = '';
     }
+    $export_where = !empty($_SESSION['export_where']) ? $_SESSION['export_where'] : '';
+    if (empty($_REQUEST['export_where_md5']) || $_REQUEST['export_where_md5'] != md5($export_where)) {
+        $err = translate('LBL_ERROR_EXPORT_WHERE_CHANGED', 'SecurityGroups');
+        SugarApplication::appendErrorMessage($err);
+        header("Location: index.php?action={$_POST['return_action']}&module={$_POST['return_module']}");
```

Comment:
I believe the preferred function here is actually:

```php
SugarApplication::redirect();
```

```suggestion
        SugarApplication::redirect("Location: index.php?action={$_POST['return_action']}&module={$_POST['return_module']}");
```